ports/espressif/boards/deneyap_kart_1a/sdkconfig

```diff
@@ -0,0 +1,19 @@
+CONFIG_ESP32_SPIRAM_SUPPORT=y
+#
+# SPI RAM config
+CONFIG_SPIRAM_TYPE_AUTO=y
+# CONFIG_SPIRAM_TYPE_ESPPSRAM16 is not set
+# CONFIG_SPIRAM_TYPE_ESPPSRAM32 is not set
+# CONFIG_SPIRAM_TYPE_ESPPSRAM64 is not set
+CONFIG_SPIRAM_SIZE=8388608
+CONFIG_SPIRAM_SPEED_80M=y
+CONFIG_SPIRAM=y
+CONFIG_SPIRAM_BOOT_INIT=y
+CONFIG_SPIRAM_IGNORE_NOTFOUND=y
+CONFIG_SPIRAM_USE_MEMMAP=y
+# CONFIG_SPIRAM_USE_CAPS_ALLOC is not set
+# CONFIG_SPIRAM_USE_MALLOC is not set
+CONFIG_SPIRAM_MEMTEST=y
+# CONFIG_SPIRAM_ALLOW_BSS_SEG_EXTERNAL_MEMORY is not set
+# CONFIG_SPIRAM_ALLOW_NOINIT_SEG_EXTERNAL_MEMORY is not set
+CONFIG_SPIRAM_CACHE_WORKAROUND=y
```

Comment:
Missing newline at end of file here.

Reply:
pre-commit excludes the following but it seems **OK** to have newlines in these files so these can be removed

- `ports/espressif/esp-idf-config/.*`
- `ports/espressif/boards/.*/sdkconfig`

https://github.com/adafruit/circuitpython/blob/3c64b5212fe396eca4b52c26fd92d80e61deb7a8/.pre-commit-config.yaml#L11

Reply:
@microdev1 Yeah, I'd agree with that certainly for the `sdkconfig` files. Do you know if the `.csv`s in `esp-idf-config/` can have blank lines also? I would hope so.

Reply:
The have newlines now so it is fine I think.